via/services/youtube_api/models.py

```diff
@@ -0,0 +1,118 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+
+@dataclass
+class VideoDetails:
```

Comment:
All of the names of these things are chosen for minimum mapping from the YouTube JSON names. I followed the format very closely.